linetools/guis/spec_widgets.py

```diff
@@ -467,6 +471,23 @@ def on_key(self, event):
                             mssg = 'Using '+ aline.__repr__()
                             mssg = mssg + ' ::  Rest EW = {:g} +/- {:g}'.format(
                                 aline.attrib['EW'].to(mAA), aline.attrib['sig_EW'].to(mAA))
+                        else:  # Faux Lya line
+                            llist = LineList('ISM')
+                            wrest = 1215.6700*u.AA
+                            wcen = np.mean(iwv)
+                            z = wcen/wrest - 1.
+                            #QtCore.pyqtRemoveInputHook()
+                            #pdb.set_trace()
+                            #QtCore.pyqtRestoreInputHook()
+                            dline = AbsLine(wrest, linelist=llist, z=z.value)
+                            tspec = XSpectrum1D.from_tuple((self.spec.wavelength, self.spec.flux, self.spec.sig))
+                            tspec.normalize(lconti)
+                            dline.analy['spec'] = tspec
+                            dline.limits.set(iwv)
+                            dline.measure_ew()
+                            mssg = 'Using dummy '+ dline.__repr__()
```

Comment:
This is so a user can measure the EW without setting a line.
Have made a small edit.